src/app/app.less

```diff
@@ -5,8 +5,3 @@
 @import 'mail-repository/header/inbox-james-mail-repository-header';
 @import 'mail-repository/email/inbox-james-mail-repository-email-display';
 @import 'mail-repository/email/inbox-james-mail-repository-email-dialog';
-@import 'deleted-messages/restore-request/inbox-james-deleted-messages-restore-request';
-@import 'deleted-messages/criteria/item/inbox-james-deleted-messages-criteria-item';
-@import 'deleted-messages/criteria/form/inbox-james-deleted-messages-criteria-form';
-@import 'deleted-messages/criteria/participant/inbox-james-deleted-messages-criteria-participant-form';
-@import 'deleted-messages/criteria/date/inbox-james-deleted-messages-criteria-date-form';
```

Comment:
I've created an issue to bring back the "Recover deleted messages" feature: https://github.com/OpenPaaS-Suite/esn-frontend-inbox/issues/146.